pkgs/servers/sql/postgresql/ext/apache_datasketches.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, lib, fetchFromGitHub, gcc, postgresql, boost182, nixosTests }:
+let
+  cpp_sources = fetchFromGitHub {
+    owner  = "apache";
+    repo   = "datasketches-cpp";
+    rev = "refs/tags/4.1.0";
+    hash = "sha256-vPoFzRxOXlEAiiHH9M5S6255ahzaKsGNYS0cdHwrRYw=";
+  };
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "apache_datasketches";
+  version = "1.6.0";
+
+  src = fetchFromGitHub {
```

Comment:
Since you have multiple sources, you maybe should think about using `srcs` instead of `src`.

Reply:
I think i got this working with `srcs` now.